internal/functions/efgs/database.go

```diff
@@ -0,0 +1,150 @@
+package efgs
+
+import (
+	"context"
+	"errors"
+	"github.com/covid19cz/erouska-backend/internal/logging"
+	"net"
+
+	"github.com/GoogleCloudPlatform/cloudsql-proxy/proxy/proxy"
+	"github.com/covid19cz/erouska-backend/internal/secrets"
+	"github.com/go-pg/pg/v10"
+	"github.com/go-pg/pg/v10/orm"
+)
+
+//Singleton database struct. Must be initialize with CreateDatabaseConnection!
+var database Database
+
+//Database Contains database connection pool.
+type Database struct {
+	connection *pg.DB
+}
+
+//CreateDatabaseConnection Create new database connection pool. Credentials must be specified in evn variables.
```

Comment:
Typo: -> `env`  
And it's not even true because you're loading the credentials from Secrets Manager :-)